src/index.js

```diff
@@ -361,4 +427,11 @@ class Dayjs {
   }
 }
 
-export default config => (new Dayjs(config))
+dayjs.extend = (plugin, option) => {
+  plugin(option, Dayjs, dayjs)
+  return dayjs
+}
```

Comment:
What advantage does using `dayjs.extend` give users over just adding to the dayjs prototype directly?

Reply:
We call this a plugin system. 😁 Just like jQuery.extend()

This just give the ability to extend our core library. Since we can't put everything into our core code.

Reply:
You can see my comment [here](https://github.com/xx45/dayjs/pull/83#issuecomment-387805390)

We will have some Day.js extensions with official support. And with out `extend`, it will be so wired like this in our document.
```javascript
// if you want to use advancedFormat, follow the instruction
import { Dayjs } from 'dayjs'
import { advancedFormat } from 'dayjs/plugin'
Dayjs.prototype.advancedFormat = advancedFormat
dayjs.advanced = () => {......}
// that all
```
Or:
```javascript
dayjs.extend(advancedFormat)
```

Reply:
That makes sense :)